report.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sbn
+import json
+
+
+class ReportAssistant():
+    def __init__(self):
+        self.usage = pd.read_csv("./state/metrics/usage.csv")
+        # self.messages = pd.read_csv("./state/metrics/messages.csv")
+        self.feedback = pd.read_csv("./state/metrics/feedback.csv")
+        self.current_df = None
+
+    def read_df(self, df: pd.DataFrame, n=10):
+        print(df.head(n))
```

Comment:
Debugging?

Reply:
Yes, I was debugging but moved to the jupyter notebook, so the report stuff is going to be updated